Tests/XcbeautifyLibTests/RendererTests/GitHubActionsRendererTests.swift

```diff
@@ -0,0 +1,676 @@
+#endif
+    }
+
+    func testCompileStoryboard() {
+        let formatted = logFormatted("CompileStoryboard /Users/admin/MyApp/MyApp/Main.storyboard (in target: MyApp)")
+        XCTAssertEqual(formatted, "[MyApp] Compiling Main.storyboard")
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCompileWarning() {
+        let input = "/path/file.swift:64:69: warning: 'flatMap' is deprecated: Please use compactMap(_:) for the case where closure returns an optional value"
+        let output = "::warning file=/path/file.swift,line=64,col=69::'flatMap' is deprecated: Please use compactMap(_:) for the case where closure returns an optional value\n\n"
+        XCTAssertEqual(logFormatted(input), output)
+        XCTAssertEqual(parser.outputType, .warning)
+    }
+
+    func testCompileXib() {
+        let input = "CompileXIB /path/file.xib (in target 'MyApp' from project 'MyProject')"
+        let output = "[MyApp] Compiling file.xib"
+        XCTAssertEqual(logFormatted(input), output)
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCopyHeader() {
+        let input = "CpHeader /path/to/destination/file.h /path/file.h (in target 'MyApp' from project 'MyProject')"
+        let output = "[MyApp] Copying file.h"
+        XCTAssertEqual(logFormatted(input), output)
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCopyPlist() {
+        let input = "CopyPlistFile /path/to/destination/file.plist /path/file.plist (in target 'MyApp' from project 'MyProject')"
+        let output = "[MyApp] Copying file.plist"
+        XCTAssertEqual(logFormatted(input), output)
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCopyStrings() {
+        let input = "CopyStringsFile /path/to/destination/file.strings /path/file.strings (in target 'MyApp' from project 'MyProject')"
+        let output = "[MyApp] Copying file.strings"
+        XCTAssertEqual(logFormatted(input), output)
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCpresource() {
+        let input = "CpResource /path/to/destination/file.ttf /path/file.ttf (in target 'MyApp' from project 'MyProject')"
+        let output = "[MyApp] Copying file.ttf"
+        XCTAssertEqual(logFormatted(input), output)
+        XCTAssertEqual(parser.outputType, .task)
+    }
+
+    func testCursor() {
+    }
+
+    func testExecuted() throws {
+        let input1 = "Test Suite 'All tests' failed at 2022-01-15 21:31:49.073."
+        let input2 = "Executed 3 tests, with 2 failures (1 unexpected) in 0.112 (0.112) seconds"
+
+        let input3 = "Test Suite 'All tests' passed at 2022-01-15 21:33:49.073."
+        let input4 = "Executed 1 test, with 1 failure (1 unexpected) in 0.200 (0.200) seconds"
+
+        // First test plan
+        XCTAssertNil(parser.summary)
+        XCTAssertFalse(parser.needToRecordSummary)
+        let formatted1 = logFormatted(input1)
+#if os(macOS)
+        // FIXME: Failing on Linux
```

Comment:
Is this in scope for this PR?

Reply:
I think this should be a separate PR, since this is copied from `TerminalRendererTests` and also an issue there. I can look into why this is triggering an error and fix for all `OutputRendering` types.